tools/loader/ImageClassifier.cpp

```diff
@@ -218,6 +222,24 @@ static void printTopKPairs(const std::vector<FloatIndexPair> &topKPairs) {
   }
 }
 
+/// Checks if /p topKPairs have the index that matches the provided index
+static int matchIndices(const std::vector<FloatIndexPair> &topKPairs,
+                        const std::string &fileName) {
+  // loop through pairs and try to match the index from file name
+  for (const auto& p : topKPairs) {
+    if (p.second - labelOffset == expectedCategoryIndex) {
```

Comment:
So it looks like if you don't provide the expectedCategoryIndex argument this will mostly fail (since it defaults to index 0). Can we make it instead so that it always passes if you don't provide the arg, that way default behavior of image-classifier is unchanged.

Reply:
Looks like this is what is breaking the CI runs for this PR.

Reply:
Fixed in last commit